libs/blocks/src/lib/hero/content-less/index.tsx

```diff
@@ -22,7 +22,7 @@ const ContentLess = ({
         className,
       )}
     >
-      <FluidContainer className="flex flex-col gap-gap-3xl">
+      <FluidContainer className="flex flex-col gap-gap-2xl">
```

Comment:
are you sure about this change? 
please check it with hero / content-less block in figma 🙏